src/generic/course-stepper/CouseStepper.scss

```diff
@@ -0,0 +1,81 @@
+@keyframes spin {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
```

Comment:
We already have an [animation to rotate](https://github.com/raccoongang/frontend-app-course-authoring/blob/2u-master/src/assets/scss/_animations.scss) the element. I suggest using it 🥇